app_wallabag2_fr.md

```diff
@@ -0,0 +1,30 @@
+## <img src="https://wallabag.org/user/themes/boxify/img/logo-wallabag-black.svg">
```

Comment:
Do not gather images from other places.
You cloud [take this one](https://yunohost.org/images/Logo-wallabag-svg.svg)
You could also add a new image on `images` folder.